content/general/concepts/cicd-pipeline/cicd-pipeline.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'CICD Pipeline'
+Description: 'CICD Pipeline automates the software / application delivery process.'
+
+Subjects: 
+  - 'CI'
+  - 'CD'
+  - 'The Other CD'
+  - 'Steps in the Pipeline'
+Tags:
+  - 'CICD'
```

Comment:
Tags should come from [this](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) list.

Reply:
Same for the tags, I'll check these out!